tests/bearlib/aspects/ClassTest.py

```diff
@@ -35,6 +36,33 @@ class docs:
         assert not SubAspect.docs.check_consistency()
 
 
+class AspectlistClassTest:
```

Comment:
Better name it `aspectlistTest` or the `aspectsYEAHBear` will get angry in the future ;)

I know... I also have to change that at several points in my existing aspects code :)